.github/workflows/build_and_test.yml

```diff
@@ -91,3 +89,106 @@ jobs:
           path: |
             ~/${{ env.RESULT_PATH }}/results.html
             ~/${{ env.RESULT_PATH }}/results.xml
+
+  integration_test:
+    name: integration_test
+    # Running on Ubuntu 18.04 as Collectd python plugin required version GLIBC_2.29 which is not available on ubuntu-latest (20.04)
+    runs-on: ubuntu-18.04
+    needs: [build]
+    strategy:
+      matrix:
+        group: [1, 2, 3, 4]
+      fail-fast: false
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v2
+
+      - name: Downloading image
```

Comment:
Can you try testing with the `bundle` artifact from https://github.com/signalfx/signalfx-agent/blob/main/.github/workflows/build_and_test.yml#L42 instead of the image?  We can try to save some time by not having to download and extract the bigger image.

Reply:
Yes, I have tried with `bundle` artifact, we can use it instead of `image` to save some time.
I have made changes to that.